### PR TITLE
[RM Ch08] Typo corrections

### LINKED ISSUE #2130: [RM Ch08] Typo's in ToC and Table 8.5
ToC: indentation of subsections is incorrect

Table 8.5 conversion from html to md has introduced some errors.

### LINKED ISSUE #2139: [RM Ch08] Correct Typo's Add clarifications to 8.3
Correct Typo's in ToC and add clarifications to [8.3 Telco Edge Cloud](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter08.md#8.3)

### PR DESCRIPTION
**Pull request by pgoyal01**
Closes #2130 
Closes #2139 

Corrected ToC and Table 8.5

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

File: doc/ref_model/chapters/chapter08.md [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
   * [8.2.2. Telco Cloud](#8.2.2)
   * [8.2.3. Telco Operator Platform Conceptual Architecture](#8.2.3)
- * [8.3. Telco Edge Cloud](#8.3)
+* [8.3. Telco Edge Cloud](#8.3)
   * [8.3.1. Telco Edge Cloud Deployment Environment Characteristics](#8.3.1)
   * [8.3.2. Telco Edge Cloud Infrastructure Characteristics](#8.3.2)
   * [8.3.3. Telco Edge Cloud Infrastructure Profiles](#8.3.3)
-  * [8.3.4. Telco Edge Cloud Infrastructure Characteristics](#8.3.4)
-  * [8.3.5. Comparison of Edge terms from various Open Source Efforts](#8.3.5)
+  * [8.3.4. Telco Edge Cloud: Platform Services Deployment](#8.3.4)
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter08.md:15]: It's nit picking, but it seems inconsistent to use ":" here and nowhere else in this list.

File: doc/ref_model/chapters/chapter08.md [hunk 2]
```diff
@@ -129,22 +129,24 @@ Based on the infrastructure deployed at the edge, the Table 8-3 specifies the [I
...
+| infra.hw.cpu.cfg.002 | Minimum Number of cores per CPU | This determines the number of cores needed per CPU. | 20 | 20 | 2 | 1 |
+| infra.hw.cpu.cfg.003 | NUMA alignment | NUMA alignment support and BIOS configured to enable NUMA | N | Y | N | Y* |
+
+- Note: immaterial if the number of CPU sockets (infra.hw.cpu.cfg.001) is 1
 
+Please note that none of the listed parameters form part of a typical OpenStack flavour except that the vCPU and memory requirements of a flavour cannot exceed the available hardware capacity.
 
 <a name="8.3.4"></a>
-### 8.3.4  Telco Edge Cloud Infrastructure Characteristics
+### 8.3.4  Telco Edge Cloud: Platform Services Deployment
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter08.md:146]: Same as above.

File: doc/ref_model/chapters/chapter08.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter08.md:15]: @ildikov Thanks - Done

File: doc/ref_model/chapters/chapter08.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/chapter08.md:146]: @ildikov Thanks - Done

**Review by ulikleber — APPROVED** [review]

**Review by ildikov — APPROVED** [review]
Thanks @pgoyal01 for the fixes.

**Review by sukhdevkapur — APPROVED** [review]

**Pull request merged**